ext/luawrapper/include/LuaContext.hpp

```diff
@@ -98,7 +105,17 @@ class LuaContext {
     explicit LuaContext(bool openDefaultLibs = true)
     {
         // luaL_newstate can return null if allocation failed
+#if defined(__SANITIZE_THREAD__) && defined(LUAJIT_VERSION) && defined(__aarch64__)
+        // on arm64 with luajit and TSAN, allocation may fail spuriously, so keep retrying a bit
+        int tries = 100;
+#else
+        int tries = 1;
+#endif
+        for (; tries > 0; tries--) {
         mState = luaL_newstate();
+        if (mState != nullptr) { break; }
```

Comment:
Could you fix the indent while you're there?
```suggestion
          mState = luaL_newstate();
          if (mState != nullptr) { break; }
```